manim/animation/transform.py

```diff
@@ -238,6 +262,30 @@ def __init__(
 
 
 class CounterclockwiseTransform(Transform):
+    """Transforms Mobjects while moving them in Counter-Clockwise direction.
+
+    Examples
+    --------
+
+    .. manim:: CounterclockwiseTransform1
+        :quality: medium
+
+        class CounterclockwiseTransform1(Scene):
+            def construct(self):
+                self.play(CounterclockwiseTransform(Square(), Circle()))
```

Comment:
Same comment as above: what would you like to illustrate with this example?

Reply:
if I used a square instead of a text, it shrinks in while transforming counter-clockwise. I just wanted to show manim users what `CounterclockwiseTransform` can do to shapes